src/api/account.js

```diff
@@ -23,26 +22,6 @@ export const getCurrentBalance = async (account) => {
   return balanceValue
 }
 
-/**
- * Returns the amount of ether tokens
- * @param {*string} account address
- */
-export const getEtherTokens = async (account) => {
-  const gnosis = await api.getGnosisConnection()
-
-  if (gnosis && gnosis.etherToken) {
-    const balance = await gnosis.etherToken.balanceOf(account) // balance is a BigNumber
-    return new Decimal(balance.toFixed(0))
-  }
-  return undefined
-}
-
-export const getOlympiaTokensByAccount = async (account) => {
-  const gnosis = await api.getGnosisConnection()
-  const balance = await gnosis.olympiaToken.balanceOf(account)
-  return new Decimal(balance.toFixed(0)).toString()
-}
-
 export const getMainnetAddressForRinkebyAccount = async (account) => {
   const gnosis = await api.getGnosisConnection()
   const address = await gnosis.olympiaAddressRegistry.mainnetAddressFor(hexWithPrefix(account))
```

Comment:
This should be configurable too. Wdyt?

Reply:
Will do it later in user registration and reward claiming ticket @andre-meyer